ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/site/js/textinputview.js

```diff
@@ -72,10 +72,16 @@
             }
             this.widget.addEventListener('blur', (e) => {
                 this._model.value = e.target.value;
+                if(this._model.displayValue) {
+                    this.widget.value = this._model.displayValue;
+                }
                 this.setInactive();
             });
             this.widget.addEventListener('focus', (e) => {
                 this.setActive();
+                if(this._model.displayValue) {
```

Comment:
You are checking for displayValue and setting the value. Also, create a re-usable function for this and re-use, why is the same code copy/pasted at multiple places